### ISSUE TITLE
adding multiple range

### ISSUE THREAD

**Issue by Akidou** [angryziber/ipscan#96]
Hi, angryziber,
did you think of adding a feature for scaning multiple ip address from a multiline text box like :
100.100.100.1-100.100.200.255
10.10.10.0-10.10.20.120
we can add any different port, but why not adding different range.

your program is very silent and stable :) and with java it become even fast.

**Comment by zero77** [comment]
Yes, it would be nice if we could do this by listing the address ranges in a text file.

**Comment by omihut** [comment]
Advanced feeder mentioned in documentation (http://angryip.org/documentation) seems to be available to do exactly that.  However, it's not clear how to select advanced feeder.  My drop down list has only 3 options, not 4 as documentation shows.
